course/style.css

```diff
@@ -0,0 +1,429 @@
+	margin-top: calc(7.5rem - var(--wp--custom--nav-mobile-gap));
+}
+
+/* Separate hover behavior for navbar button, applicable when in mobile view, as the colors are separate for mobile view */
+.wp-block-navigation .wp-block-navigation__responsive-container.is-menu-open .wp-block-navigation-link.is-style-navigation-link-button a:hover {
+	background-color: var(--wp--preset--color--background);
+	color: var(--wp--preset--color--foreground);
+}
+
+header.wp-block-template-part {
+	background-color: transparent;
+	padding: 2.75rem 0px;
+}
+
+/*
+ * Comments
+ */
+.wp-block-post-comments-form form.comment-form p input:not([type=submit]),
+.wp-block-post-comments-form form.comment-form textarea {
+	border: 1px solid var(--wp--preset--color--foreground);
+	border-radius: 8px;
+	background-color: var(--wp--preset--color--background);
+}
+
+.wp-block-post-comments-form .comment-form-cookies-consent label {
+	margin: auto 0;
+}
+
+.wp-block-post-comments-form .comment-form {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	grid-column-gap: var(--wp--custom--reply-column-gap);
+	grid-row-gap: var(--wp--custom--comment-gap-medium);
+}
+
+@media screen and (max-width: 480px) {
+	.wp-block-post-comments-form .comment-form {
+		display: flex;
+		flex-direction: column;
+		gap: calc(var(--wp--preset--spacing--60) - var(--wp--preset--spacing--40));
+	}
+}
+
+.wp-block-post-comments-form .comment-form p,
+.wp-block-post-comments-form h3 {
+	margin-block-start: 0;
+	margin-block-end: 0;
+}
+
+.wp-block-post-comments-form .comment-form .comment-notes {
+	display: none;
+}
+
+.wp-block-post-comments-form .comment-form .comment-form-author {
+	order: 1;
+	grid-column: 1 / 3;
+}
+
+.wp-block-post-comments-form .comment-form .comment-form-email {
+	order: 2;
+	grid-column: 1 / 2;
+}
+
+.wp-block-post-comments-form .comment-form .comment-form-url {
+	order: 3;
+	grid-column: 2 / 3;
+}
+
+.wp-block-post-comments-form .comment-form .comment-form-comment {
+	order: 4;
+	grid-column: 1 / 3;
+}
+
+.wp-block-post-comments-form .comment-form .comment-form-cookies-consent {
+	order: 5;
+	grid-column: 1 / 3;
+}
+
+.wp-block-post-comments-form .comment-form .wp-block-button {
+	order: 6;
+	grid-column: 1 / 2;
+}
+
+.wp-block-comments-query-loop .wp-block-comment-template .wp-block-comment-author-name {
+	margin-bottom: var(--wp--custom--comment-gap-small);
+}
+
+.wp-block-comments-query-loop .wp-block-comment-template .wp-block-comment-content p {
+	margin-block-start: var(--wp--custom--comment-gap-medium);
+	margin-block-end: var(--wp--custom--comment-gap-small);
+}
+
+.wp-block-comments-query-loop .comment > .is-vertical > .wp-block-group {
+	align-items: flex-start;
+	gap: var(--wp--custom--comment-gap-small);
+}
+
+.wp-block-comments-query-loop .comment > div > div > .is-vertical {
+	gap: 0;
+}
+
+/*
+ * Link styles
+ */
+a {
+	text-decoration-thickness: .0625em !important;
+	text-underline-offset: .15em
+}
+
+/*
+ * Search styles
+ */
+.wp-block-search__button {
```

Comment:
What's the reason for setting this in CSS? 

This is what the default search button looks like when I insert a search block:
 
<img width="658" alt="Screenshot 2023-03-03 at 11 27 14 AM" src="https://user-images.githubusercontent.com/5375500/222774269-76ed68aa-8045-4236-a3b1-53a8025b9d87.png">

Reply:
I wasn't able to reproduce that issue, but when I removed the CSS it didn't appear to break anything either - [1bacf4a](https://github.com/Automattic/themes/pull/6888/commits/1bacf4a63262fc9e7f5d3d72a551886b5609284a).